docs/api/README.md

```diff
@@ -514,7 +514,8 @@ Accept: application/json
         "property2": {
           "admin": "true"
         }
-      }
+      },
+      "force": true
```

Comment:
not sure it's worth the change, but I'd put it to false here to show it's the default (and as forcing something is dangerous, you don't want people to copy/paste this without knowing what that param does)